client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -544,10 +544,16 @@ export default {
     setInitialValues () {
       this.values = { ...this.initValues }
       // Set default values to ensure reactivity.
-      if (typeof this.values.submitter === 'undefined' || Object.keys(this.values.submitter).length === 0) {
+      if (typeof this.values.submitter !== 'undefined' && typeof this.values.submitter.institution === 'undefined') {
+        // Since Data sends us the key toeb instead of institution, we need to transform this for now but keep all init values
+        Vue.set(this.values.submitter, 'institution',  this.values.submitter.toeb)
+        Vue.delete(this.values.submitter, 'toeb')
```

Comment:
in preparation for vue3 can we omit the Vue.set/delte?
```suggestion
        this.values.submitter.institution = this.values.submitter.toeb
        delete this.values.submitter.toeb
```

Reply:
doing it like this will make `values.submitter.institution` non-reactive and the trigger to render the additional form fields will not be fired. However in Vue3 this problem does not exist anymore and we can set the data props as usual to make them reactive. Nevertheless, this syntax is not supported in vue3 but in vue3's compat build. This component is also a candidate for a bigger refactoring which I do not want to start before EWM v3.0 is deployed.